packages/tempus/src/Tempus_StepperNewmarkExplicitAForm_impl.hpp

```diff
@@ -276,6 +307,12 @@ void StepperNewmarkExplicitAForm<Scalar>::takeStep(
     predictDisplacement(*d_new, *d_old, *v_old, *a_old, dt);
     predictVelocity(*v_new, *v_old, *a_old, dt);
 
+    stepperNewmarkExpAppAction_->execute(solutionHistory, thisStepper,
+        StepperNewmarkExplicitAFormAppAction<Scalar>::ACTION_LOCATION::BEFORE_SOLVE);
+
+    stepperNewmarkExpAppAction_->execute(solutionHistory, thisStepper,
+        StepperNewmarkExplicitAFormAppAction<Scalar>::ACTION_LOCATION::AFTER_SOLVE);
+
```

Comment:
Let's go with BEFORE_EXPLICIT_EVAL and AFTER_EXPLICIT_EVAL around the this->evaluateExplicitODE(a_new, d_new, v_new, time_old, p) call. There is no solve in the explicit Newmark.

Reply:
I tried to stay consistent with ERK. There's no solve there either but similar call is made. What do you think?

Reply:
So we made the AppAction consistent across ERK, DIRK and IMEX, because we can "derive" ERK and DIRK from IMEX, so to allow RKAppActions to be usable between all the RK methods, they all have all the RKAppAction locations.  However to do that across all steppers seemed excessive and possibly confusing.  So the other steppers have more tailored AppActions and likely have AppAction locations that others do not.  If anything this stepper, Newmark explicit, would look closer to ForwardEuler, which has BEGIN_STEP, BEFORE_EXPLICIT_EVAL, and END_STEP.  I thought AFTER_EXPLICIT_EVAL might be useful because of the correctVelocity and getUseFSAL block.  Does this seem reasonable?

Reply:
That makes sense. Thank you.